typings/objection/index.d.ts

```diff
@@ -1442,7 +1442,7 @@ declare namespace Objection {
     QueryBuilder: typeof QueryBuilder;
 
     tableName: string;
-    idColumn: string | string[];
+    idColumn: null | string | string[];
```

Comment:
Should probably be last, not first? Also, it was suggested here to use never instead of null? What's more suitable?

https://github.com/Vincit/objection.js/issues/2693#issuecomment-2683825272

Reply:
I don't get this `never` thing when the doc explicitly says to use `null`

![image](https://github.com/user-attachments/assets/ead104d9-aa65-451b-9073-1399f8163ee6)

Reply:
> Should probably be last, not first? Also, it was suggested here to use never instead of null? What's more suitable?
> 
> [#2693 (comment)](https://github.com/Vincit/objection.js/issues/2693#issuecomment-2683825272)

I don't think the order matters. Just let me know if you want me to change it